```
Hit:18 https://packagecloud.io/github/git-lfs/ubuntu focal InRelease
Err:48 https://dl.bintray.com/sbt/debian  Release
  502  Bad Gateway [IP: 52.43.14.61 443]
Reading package lists...
E: The repository 'https://dl.bintray.com/sbt/debian  Release' no longer has a Release file.
Error: Process completed with exit code 100.
```

big oof

--- a/lib/libshout-idjc/CMakeLists.txt
+++ b/lib/libshout-idjc/CMakeLists.txt
@@ -75,12 +75,10 @@ target_link_libraries(${PROJECT_NAME} PUBLIC Ogg::ogg)
 find_package(Vorbis REQUIRED)
 target_link_libraries(${PROJECT_NAME} PUBLIC Vorbis::vorbis Vorbis::vorbisenc)
 
-# Optional system dependencies
-find_package(OpenSSL)
-if(TARGET OpenSSL::SSL)
-    target_compile_definitions(${PROJECT_NAME} PRIVATE HAVE_OPENSSL)
-    target_link_libraries(${PROJECT_NAME} PRIVATE OpenSSL::SSL OpenSSL::Crypto)
-endif()
+# OpenSSL
+find_package(OpenSSL REQUIRED)
+target_compile_definitions(${PROJECT_NAME} PRIVATE HAVE_OPENSSL)
+target_link_libraries(${PROJECT_NAME} PRIVATE OpenSSL::SSL OpenSSL::Crypto)
 
 option(SPEEX "Speex support" OFF)
 if(SPEEX)

--- a/tools/debian_buildenv.sh
+++ b/tools/debian_buildenv.sh
@@ -74,6 +74,7 @@ case "$COMMAND" in
             libsndfile1-dev \
             libsoundtouch-dev \
             libsqlite3-dev \
+            libssl-dev \
             libtag1-dev \
             libupower-glib-dev \
             libusb-1.0-0-dev \

